spec/models/mhv_account_spec.rb

```diff
@@ -247,11 +247,12 @@
 
     context 'existing account that has not been upgraded' do
       let(:mhv_ids) { ['14221465'] }
-      let(:base_attributes) { { user_uuid: user.uuid, account_state: 'unknown' } }
+      let(:base_attributes) { { user_uuid: user.uuid, account_state: 'registered' } }
 
       it 'will only upgrade an account and set the time the account was upgraded' do
+        subject = described_class.new(base_attributes.merge(registered_at: Time.current))
```

Comment:
How come I have to declare `registered_at` with this merge? I tried adding it to the hash above at first, but it didn't work until I used `merge`.

Reply:
When vets.gov creates an account, we also set the time the account is registered_at. Similarly, we set upgraded_at when the account gets upgraded. I don't think we have validations on the model though, so it must be the spec that is testing for this.